go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/arduino/libraries-repository-engine
 
-go 1.14
+go 1.16
```

Comment:
I realize this change does not necessarily follow from the others, but it seems that in this case there are only downsides to indicating compatibility with a Go version we don't do any testing for. It's likely that `arduino/library-registry` is the only user of the module anyway, since its API currently only provides for reading the data format of the library registry file and that program has already been updated to Go 1.16.